performance.php

```diff
@@ -4,3 +4,4 @@
 require_once( __DIR__ . '/performance/bulk-edit.php' );
 require_once( __DIR__ . '/performance/vip-tweaks.php' );
 require_once( __DIR__ . '/performance/do-pings.php' );
+require_once( __DIR__ . '/performance/plugins.php' );
```

Comment:
Should we use `glob( __DIR__ . '/performance/*.php' )` here to include all files in the performance directory?